framework/src/modules/chain/submodules/loader.js

```diff
@@ -741,36 +741,31 @@ __private.validateOwnChain = cb => {
  * Snapshot creation - performs rebuild of accounts states from blockchain data
```

Comment:
Please, make another round to check for `snapshot` matches across the codebase. Careful with the concept of round snapshot which is different is meant to stay.

Reply:
Obsolete and removed (with test), thanks!